lib/api/core/janitor.js

```diff
@@ -186,6 +186,8 @@ class Janitor {
    * @returns {Promise}
    */
   dump (suffix) {
+
+    return Bluebird.resolve();
     if (this._dump) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this code after the "return" statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AUnreachableCode)